Sources/AlgoliaSearchClient/Models/Search/Response/SearchResponse/Auxiliary/RenderingContent/FacetOrderContainer.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  FacetOrderContainer.swift
+//  
+//
+//  Created by Vladislav Fitc on 17/03/2021.
+//
+
+import Foundation
+
+public struct FacetOrderContainer: Codable {
```

Comment:
Is this still used? is`RenderingContent` enough?

Reply:
Good catch, it's obsolete. Removed it.